spec/hydra/pcdm/models/collection_spec.rb

```diff
@@ -26,9 +26,13 @@
         subject.members << object1
         subject.members << object2
         subject.members << collection3
+        subject.save
```

Comment:
> Is the call to #save necessary to get IDs?

Yeah, it gets the IDs from the persisted graphs in Fedora - there's no hasParent metadata or anything on the child object.